arb_os/output.mini

```diff
@@ -116,17 +113,10 @@ public impure func outputStats_endOfBlock(blockNum: uint, timestamp: uint) {
             statsAllTime: updatedStats
         };
 
-        let gasUsedThisBlock = globalBlockReceiptData.statsThisBlock.totalGasUsed;
-        let gasLimit = chainParams_speedLimitPerBlock();
-        if (gasLimit < gasUsedThisBlock) {
-            gasLimit = gasUsedThisBlock;
-        }
-
         asm((
             const::LogType_blockSummary,
             blockNum,
             timestamp,
-            gasLimit,
```

Comment:
The RPC interface is gonna need to use something as the block gas limit. Not sure what makes sense for that

Reply:
A good option is 13.5 times the per-second speed limit.  That will be correct on average, on a busy non-sequencer chain.